scripts/docker-compose/imports/openzaak-database/database/2-setup-melding-evenement-organiseren-behandelen-zaaktype.sql

```diff
@@ -284,9 +284,42 @@ VALUES
   NULL        -- datum_einde_geldigheid
 );
 
-
 -- For the second JSON object
 INSERT INTO catalogi_statustype
+(
+    id,
+    uuid,
+    statustype_omschrijving,
+    statustype_omschrijving_generiek,
+    statustypevolgnummer,
+    informeren,
+    statustekst,
+    toelichting,
+    zaaktype_id,
+    _etag,
+    doorlooptijd,
+    datum_begin_geldigheid,
+    datum_einde_geldigheid
+)
+VALUES
+    (
```

Comment:
It is possible to add multiple values in 1 insert statement, but it's not important here :)